cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -754,38 +754,131 @@
       multiple machines to prevent data loss in the event of a hardware
       failure. The recommended setting is to run at least two CDAP Kafka servers.
       If you are running two CDAP Kafka servers, set this value to 2; otherwise,
-      set it to the number of CDAP Kafka servers.
+      set it to the maximum number of tolerated machine failures
+      plus one (assuming you have that number of machines).
     </description>
   </property>
 
   <property>
-    <name>kafka.log.dir</name>
+    <name>kafka.server.log.dirs</name>
     <value>/tmp/kafka-logs</value>
     <description>
       CDAP Kafka service log storage directory
     </description>
   </property>
 
   <property>
-    <name>kafka.num.partitions</name>
+    <name>kafka.server.log.flush.interval.messages</name>
+    <value>10000</value>
+    <description>
+      The interval length at which will force an fsync of data written to the log.
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.log.retention.hours</name>
+    <value>24</value>
+    <description>
+      The number of hours to keep a log file before deleting it
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.num.partitions</name>
     <value>10</value>
     <description>
       Default number of partitions for a topic
     </description>
   </property>
 
   <property>
-    <name>kafka.seed.brokers</name>
-    <value>127.0.0.1:9092</value>
+    <name>kafka.server.socket.receive.buffer.bytes</name>
+    <value>1048576</value>
+    <description>
+      The size in bytes of the receive buffer the server prefers for socket connections
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.socket.send.buffer.bytes</name>
+    <value>1048576</value>
+    <description>
+      The size in bytes of the send buffer the server prefers for socket connections
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.zookeeper.connection.timeout.ms</name>
+    <value>1000000</value>
+    <description>
+      The maximum time (in milliseconds) that the client will wait to establish a connection to Zookeeper
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.socket.receive.buffer.bytes</name>
+    <value>${kafka.server.socket.receive.buffer.bytes}</value>
+    <description>
+      The size in bytes of the receive buffer the server prefers for socket connections (deprecated: replaced with
+      kafka.server.socket.receive.buffer.bytes)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.socket.send.buffer.bytes</name>
+    <value>${kafka.server.socket.send.buffer.bytes}</value>
+    <description>
+      The size in bytes of the send buffer the server prefers for socket connections (deprecated: replaced with
+      kafka.server.socket.send.buffer.bytes)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.zookeeper.connection.timeout.ms</name>
```

Comment:
oh I guess I added these because I was looking at the Kafka configs documentation.